JDBC/DeleteBooks.java

```diff
@@ -0,0 +1,52 @@
+package com.metacube.jdbc.example;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+public class DeleteBooks {
+	public static void main(String[] args) throws IOException {
+		getDeletedBooks();
+		System.out.println(DeleteBookPOJO.getNumberOfBooksDeleted()+" Book Deleted Successfully");
+	}
+	/* execute query using prepared statement*/
+	/*method to get number of books deleted*/
+	private static boolean getDeletedBooks() {
+		Connection con = null;
+		PreparedStatement ps = null;
+		ConnectionUtil conUtil = new ConnectionUtil();
+		con = conUtil.getConnection();
+		/*Query to Delete all those books which were not issued in last 1 year. Return the number of books deleted.*/
+		String query =	"DELETE FROM books WHERE "
+				+ "(books.accession_no "
```

Comment:
ACTION: Don't use + operator to concat additive String because it will up crating too many objects, String is immutable object so if modify a string value java will create a new object.You should use StringBuilder.append() function to append these many strings.you can read here [link](]http://stackoverflow.com/questions/878573/java-multiline-string?rq=1)